js/scenes/Settings.js

```diff
@@ -31,7 +31,12 @@ export default class Settings extends Component {
     });
   }
 
+  onNavigate(id) {
```

Comment:
I'd say `onNavigationChange` though to make the relation with a user interaction more obvious.